doc/governance.rst

```diff
@@ -21,14 +21,56 @@ Bodies
 Ceph Executive Council
 ======================
 
-Responsibilities
-----------------
+.. _exec-council-responsibilities:
+
+Ceph Executive Council Responsibilities
+---------------------------------------
+
+- Spokesperson
+
+   - welcome/keynote for cephalocon
+     
+   - maintaining slides and presenting about the project
+
+   - Community focal point (user interaction, conference talks, mailing list,
+     etc)
+   
+- Community
+
+    - managing community manager
+
+    - LF Program Manager person, Social Media person
+
+    - liase with the ambassadors
+
+    - make sure ceph events happen, successfully: cephalocon, ceph days, cds, user/dev, cdm
+
+    - coordinating with LF
+
+    - creating program committee
+
+    - recordings on youtube
+
+    - getting sponsors for events
+
+    - communications, schedule, venue decisions
+
+    - coordinate blog posts
+
+
+- Ceph Foundation
+
+    - ensure foundation is healthy: financials, operations
+
+    - represent the CLT on the Board
+
+    - present project status regularly (yearly)
+
+    - collect member ideas / feedback
+
+    - ensure members feel valued
```

Comment:
no no, we don't edit this doc except by consensus in the CLT meeting. This is our governance document (like a constitution).

The additions here should go in another doc that lists "responsibilities in practice". Alternatively, we can put this PR up for a vote.

Reply:
@batrick 

I'll roll this back, and we can have a short chat in the CLT in two-and-a-half hours, and we'll get it right.